Igalvan - Livingston

```diff
@@ -1,5 +1,6 @@
 Livingston
 Restaurants
+Mayrita's 
 Things to do
```

Comment:
Not much to do but enjoy nature. It's an agricultural area, lots of orchards: almond, peach, walnut, grapes... some lakes and rivers. Social clubs... hiphop, a big Hindi and Mexican community